.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,34 +1,54 @@
-<!-- The Title above should provide a general summary of the PR -->
-<!-- Any PR which has missing info is not ready to be reviewed -->
+<!-- The title of the PR ↑↑↑ above should provide a general summary of what it is about. -->
+<!-- Warning: any PR that has missing info is not ready to be reviewed. -->
+
 # Description
 ## Issue
-<!-- Use `closes #1` with the number of the issue to find the issue -->
+<!-- Link the PR to one or more issues. -->
+<!-- Use "closes" followed by the issue ID. 
+closes #314159
+-->
 
 ## What
-<!-- What did you do? -->
-_Added a new endpoint, fixed a bug... etc._
+<!-- What did you do? (Functionally).
+Added this new feature/value, fixed that bug/behaviour…
+-->
 
 ## How
-<!-- How did you do it? (Technically) -->
-_I used this library to add this to make that better_
+<!-- How did you do it? (Technically).
+I used this library to add this to make that better.
+-->
 
 ## Related PRs
-<!-- List of related PRs against other branches/repos -->
-<!-- - [ ] #1 -->
+<!-- List of possible related PRs against other branches/repos.
```

Comment:
OK for related PR in others repos, but in others branches that shouldn't exists ...

Reply:
OK Understood.